packages/fast-components-styles-msft/src/heading/index.ts

```diff
@@ -1,36 +1,27 @@
-import { DesignSystem } from "../design-system";
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
 import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
 import { HeadingClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
-import { fontWeight } from "../utilities/fonts";
+import { applyFontWeight } from "../utilities/fonts";
 
-function applyHeadingStyles(): CSSRules<DesignSystem> {
-    return {
-        fontWeight: `${fontWeight.semibold}`,
-    };
-}
-
-/**
- * TODO #306: Pull font weight styles when we have an API for font/variable font properties
- */
 const styles: ComponentStyles<HeadingClassNameContract, DesignSystem> = {
     heading: {
         "&$heading__1": {
-            ...applyHeadingStyles(),
+            ...applyFontWeight("semibold"),
```

Comment:
I think these should all be enums

Reply:
Adding @nicholasrice for his thoughts; most of the ways we are referencing interface keys is with strings. I'm open to using enums, but then we need an enum that maps to the interface, yes?

Reply:
Hmmm - yea that would be a relationship we would need to keep in sync - though I don't think that would be particularly challenging. Enums *are* nice, but you would also then need to import two dependencies to make the utility work which seems a *little* overkill.

A middle-ground might be to export named functions - `applyFontWeightBold` vs `applyFontWeightNormal` - at least then we're not passing the same string around everywhere.